app.py

```
@@ -1,420 +1,197 @@
-from flask import Flask, render_template, request, redirect, url_for, jsonify, session
-import subprocess
-import psutil
-import signal
-import sys
-import atexit
+from flask import Flask, render_template, request, jsonify
+import asyncio
+from connection import Connection
+from threading import Thread
+import webbrowser
```

Ruff (0.8.2)

`webbrowser` imported but unused

Remove unused import: `webbrowser`

(F401)